.version

```diff
@@ -1 +1 @@
-18.09
+18.09
```

Comment:
Looking at the history of this .version file, there were a couple times where this has been updated to remove the trailing newline, and the reason for that is the `nixpkgs/.editorconfig` file which specifies that a trailing newline should be inserted for all files. You can add the following section to that file to prevent further newline insertions for that file:

```ini
[.version]
insert_final_newline = false
```

Reply:
can this be addressed in a different commit or PR for the git history?